convert2rhel/subscription.py

```diff
@@ -223,13 +223,17 @@ def remove_original_subscription_manager():
 
 
 def install_rhel_subscription_manager():
-    loggerinst.info("Installing subscription-manager RPMs.")
+    loggerinst.info("Checking for subscription-manager RPMs.")
```

Comment:
We're still installing subscription-manager in the method. We just have more checks to track and untrack the now installed packages

Reply:
Yeah, but the `loggerinst.info("Installing subscription-manager RPMs.")` is just at the line 236, since below that log is the actual install, I just changed this one because it makes more sense since we are checking for the availability of the rpms first, and then, installing the submr